backend/app/services/chat_service.py

```diff
@@ -70,14 +71,10 @@ async def generate_response(
                 .all()
             )
             for message in all_history_messages:
-                marker = "__LLM_RESPONSE__"
```

Comment:
What's the reasoning behind this change @wayangalihpratama ? Doesn't splitting at the marker do something important (I encountered this as part of the RAG evaluation work, can't quite remember the purpose now though)

Reply:
Hi @jcbashdown,

This part was added by me and is not originally from the RAG Web UI.
When I added it, I realized we already have the same code in another part of the file:
[Lines 260–264](https://github.com/akvo/akvo-rag/blob/35962df2da6138e93cf29f2fc8c36a76218b5279/backend/app/services/chat_service.py#L260-L264) from the original RAG Web UI source code.
That’s why I decided to delete it.